src/www/system_general.php

```diff
@@ -83,6 +84,19 @@
         $input_errors[] = gettext("The domain may only contain the characters a-z, 0-9, '-' and '.'.");
     }
 
+    /* Domain search list Validation 256 char length max */
+    if (!empty($pconfig['domainsearchlist'])) {
+      if (strlen($pconfig['domainsearchlist']) > 256) {
+        $input_errors[] = gettext("Custom domain search list can not exceed 256 char length");
+      } else {
+        foreach (explode(' ',$pconfig['domainsearchlist']) as $searchdomain) {
+          if (!is_domain($searchdomain)) {
```

Comment:
The DHCPv4 services uses semicolon to separate domain names (see /services_dhcp.php). We should aim in consistency.

```suggestion
        foreach (explode(';', $pconfig['domainsearchlist']) as $searchdomain) {
          if (!is_domain(trim($searchdomain))) {
```

Reply:
I wasn't aware of this syntax :-/ I always used space separated list. but let stick to the man :-)